.gitignore

```diff
@@ -2,3 +2,5 @@
 /bin/
 /libexec/
 /.vendor/
+/.cache
+/build
```

Comment:
I changed the output of the build artifacts. Now everything ends up in `./build`:

```bash
% find ./build
./build
./build/dist
./build/dist/gh-ost-binary-osx-20190531141809.tar.gz
./build/dist/gh-ost_1.0.48_amd64.deb
./build/dist/gh-ost-1.0.48-1.x86_64.rpm
./build/dist/gh-ost-binary-linux-20190531141809.tar.gz
./build/bin
./build/bin/gh-ost-linux-amd64
./build/bin/gh-ost-darwin-amd64
```

Totally get it if you don't like this. I thought I'd offer it up first though. That way we're not splitting the `./bin` and `/tmp/gh-ost-release` directories.

Reply:
`build/` makes perfect sense! Let's add `/build/` to `.gitignore`

Reply:
Will change! 

For my own education, how does git treat `/build` and `/build/` differently?